assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+/*
+Red/Teal
+rgba(204, 40, 41, 1), #4ba58c
+*/
+/*
+Grey-Blue
+
+*/
 html, body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: helvetica, sans-serif;
+  font-family: 	"mr-eaves-xl-modern", helvetica;
+  font-weight:400;
   line-height: 1.3;
-  color: #2b2b2b;
+  color:rgba(37,52,62, 1);
   background: #FBFCFC;
   font-size: 110%;
 }
 
-a {
-  color: #F1853D;
-  text-decoration: none;
-}
-
-button {
-  background: #F1853D;
-  border: 0;
-  color: #fff;
-  padding: 13px 30px;
-  font-size: 1.35em;
-}
-
-input,textarea {
-  font-size: inherit;
-  font-family: inherit;
-  -webkit-appearance: none;
-  border-radius: 0;
-}
-
-fieldset {
-  border: 0;
-  padding: 0;
-  margin: 0;
-}
-
-fieldset[disabled], button[disabled] {
-  opacity: 0.5;
-  filter: alpha(opacity=50);
+div{
+  display:inline-block;
+  width:100%;
 }
 
 nav ul {
   list-style: none;
 }
 
-@media screen and (max-width: 650px) {
-  body {
-    font-size: 95%;
-  }
-}
+body {
+  font-size: 95%;
+}
+
+h1, h2, h3, h4, h5, h6{
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h1{
+  font-size:64px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h2{
+  font-size:48px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h3{
+  font-size:48px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+h4{
+  font-size:24px;
+  font-family: 	"mr-eaves-xl-modern";
+  font-weight:700;
+}
+
+p{
+  font-size:18px;
+}
+
+/*Top Bar*/
+
+.top-bar{
+  padding:20px 0 20px 0;margin:0;
+  width:100%;
+  display:none;
+  position:fixed;
+  top:0;left:0;
+  z-index:998;
+  background:rgba(24,34,41, 1);
+  border-bottom:1px solid rgba(47,62,72, 1);
+}
+.top-bar.after-fold{
+  display:block;
+  opacity:1;
+}
+
+.top-bar .inner-wrapper{
+  max-width:1200px;
+  display:block;
+  margin:0 auto;
+  padding:0;
+}
+.top-bar .inner-wrapper img.top-mark{
+  text-align:left;
+  color:#fff1ef;
+  margin:0;padding:0;
+  font-size:24px;
+  float:left;
```

Comment:
Try using flexbox. It might sound harder at first but it's seriously _so_ much easier. Things that were hard before are literally like 2 lines of CSS. I made some examples here: http://jsbin.com/tuluzatove/edit?html,css,output